rholang/src/main/bnfc/rholang_mercury.cf

```diff
@@ -61,18 +61,38 @@ PAnd.            Proc5  ::= Proc5 "and" Proc6 ;
 POr.             Proc4  ::= Proc4 "or" Proc5 ;
 PSend.           Proc3  ::= Name Send "(" [Proc] ")" ;
 PContr.          Proc2  ::= "contract" Name "(" [Name] NameRemainder")" "=" "{" Proc "}" ;
-PInput.          Proc2  ::= "for" "(" Receipt ")" "{" Proc "}" ;
+PInput.          Proc2  ::= "for" "(" [Receipt] ")" "{" Proc "}" ;
 PChoice.         Proc2  ::= "select" "{" [Branch] "}" ;
 PMatch.          Proc2  ::= "match" Proc4 "{" [Case] "}" ;
 PBundle.         Proc2  ::= Bundle "{" Proc "}" ;
+PLet.            Proc2  ::= "let" Decl Decls "in" "{" Proc "}" ;
 PIf.             Proc1  ::= "if" "(" Proc ")" Proc2 ;
 -- Use precedence to force braces around an interior if.
 PIfElse.      Proc1 ::= "if" "(" Proc ")" Proc2 "else" Proc1 ;
 PNew.         Proc1 ::= "new" [NameDecl] "in" Proc1 ;
+PSendSynch.   Proc1 ::= Name "!?" "(" [Proc] ")" SynchSendCont ;
```

Comment:
Just curious. Is it a typo or is there some meaning in the last letter `h`?
What I mean is the `h` at the end of `PSendSynch`.

Reply:
I think it is just short for `synchronous` which I usuaslly use `sync`. So nvm.